libs/ui/src/prose.tsx

```diff
@@ -1,6 +1,22 @@
 import styled from 'styled-components';
 import { Theme } from './themes';
 
+// Prose!
```

Comment:
Prose ultimately is a <p> element right? Can we explain that and how you shouldn't nest prose's inside of one another?